src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -564,16 +528,19 @@ Namespace Microsoft.CodeAnalysis.Operations
                 Return Create(boundConversion.Operand)
             End If
 
-            Dim type As ITypeSymbol = boundConversion.Type
-            Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundConversion.ConstantValueOpt)
-            Dim isImplicit As Boolean = boundConversion.WasCompilerGenerated OrElse Not boundConversion.ExplicitCastInCode
-
-            Dim conversionOperandAndMethod = GetConversionInfo(boundConversion)
-            Dim boundOperand As BoundExpression = conversionOperandAndMethod.Operand
-            Dim conversion As Conversion = CreateConversion(boundConversion)
+            Return CreateBoundConversionOrCastOperation(boundConversion, isTryCast:=False)
+        End Function
 
-            If boundOperand.Syntax Is boundConversion.Syntax Then
-                If boundOperand.Kind = BoundKind.ConvertedTupleLiteral AndAlso boundOperand.Type = boundConversion.Type Then
+        Private Function CreateBoundConversionOrCastOperation(boundConversionOrCast As BoundConversionOrCast, isTryCast As Boolean) As IOperation
+            Dim isChecked As Boolean = False
```

Comment:
>Dim isChecked As Boolean = False [](start = 12, length = 32)

This doesn't look right, conversions in VB can be checked. This looks like a preexisting bug, please open an issue to track it.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/23677](https://github.com/dotnet/roslyn/issues/23677).

---
In reply to: [155873408](https://github.com/dotnet/roslyn/pull/23315#discussion_r155873408) [](ancestors = 155873408)